Themes/default/css/responsive.css

```diff
@@ -562,11 +555,56 @@
 	.dropmenu ul li a {
 		width: auto !important;
 	}
+	.dropmenu li.subsections > a::after {
+		position: absolute;
+		padding: 5px 0;
+		right: 10px;
+		font: 83.33%/150% Arial, sans-serif;
+		content: "\25bc" !important;
+	}
 	.dropmenu li ul,
 	.dropmenu li li:hover ul, .dropmenu li li ul {
 		position: relative;
 		border-radius: 0;
 		left: 0;
+		box-shadow: none;
+	}
+	/* 3rd level menu tests */
+	.dropmenu li ul ul {
+		margin: 0 !important;
+	}
+}
+
+@media (min-width: 480px) and (max-width: 560px) {
```

Comment:
Should be min-width **481**

Reply:
That's been wrong for a long time...

Fixed in 697406e.